code/modules/clothing/head/helmet.dm

```diff
@@ -456,14 +458,23 @@
 	if(policy)
 		to_chat(magnification, policy)
 	icon_state = "[icon_state]up"
+	REMOVE_TRAIT(magnification, TRAIT_PRIMITIVE, SPECIES_TRAIT) //Monkeys with sentience should be able to use less primitive tools.
 
 /obj/item/clothing/head/helmet/monkey_sentience/Destroy()
-	disconnect()
+	if(magnification)
+		ADD_TRAIT(magnification, TRAIT_PRIMITIVE, SPECIES_TRAIT)
```

Comment:
I don't like these side-effects on Destroy (this and the ghostize). Destroy should be nothing but destruction. This should be on `obj_destruction`.